homeassistant/components/fritz/device_tracker.py

```diff
@@ -116,4 +116,6 @@ def extra_state_attributes(self) -> dict[str, str]:
             attrs["connection_type"] = device.connection_type
         if device.ssid:
             attrs["ssid"] = device.ssid
+        if device.speed:
+            attrs["speed"] = str(device.speed)
```

Comment:
```suggestion
            attrs["speed"] = device.speed
```

Should we add a unit for the speed?

Reply:
> Are you sure, that we can remove the conversion to string?

yeah, we need to adjust the return type annotation
```diff
+def extra_state_attributes(self) -> dict[str, str]:
-def extra_state_attributes(self) -> dict[str, str | int]:
```

> I think `Mbit/s` would be a good choice. What do you think about it?

In which unit is the value provided? do we need to convert the value, so we get MBit/s ?